packages/material-ui-benchmark/package.json

```diff
@@ -34,5 +34,8 @@
     "react-dom": "^16.13.0",
     "react-jss": "^10.3.0",
     "styled-system": "^5.1.2"
+  },
+  "devDependencies": {
+    "cross-env": "^7.0.2"
```

Comment:
Do we need to define the dependency or is https://github.com/mui-org/material-ui/blob/16047fdc31cb368adef9ea03c69a364ed94f03c9/package.json#L93 enough?

Reply:
Didn't saw that we have it on root, removed it is not necessary, reverted 👍